pom.xml

```diff
@@ -415,6 +415,20 @@
                   </manifestEntries>
                 </transformer>
               </transformers>
+              <filters>
+                <filter>
+                  <artifact>org.folio:raml-module-builder</artifact>
```

Comment:
I suppose one need to use here
`<artifact>org.folio:domain-models*</artifact>`
instead of
`<artifact>org.folio:raml-module-builder</artifact>`
 according to https://github.com/folio-org/mod-finance/pull/4#discussion_r323089371

Reply:
@khandramai - I agree we used different artifact in mod-finance but I do not see `[INFO] No artifact matching filter org.folio:raml-module-builder` message here.

Reply:
I see it in the build of your branch: https://jenkins-aws.indexdata.com/job/folio-org/job/mod-organizations-storage/job/MODORGS-42/1/consoleFull
> [INFO] No artifact matching filter org.folio:raml-module-builder

Reply:
Thanks, I was able to verify it and have pushed a new commit for it.